mirascope/core/base/types.py

```diff
@@ -0,0 +1,22 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pydub import AudioSegment
```

Comment:
won't this fail if pydub isn't installed?

Reply:
OK, I forgot `from __future__ import annotations` in this file.

Reply:
If TYPE_CHECKING: is only evaluated during type checking, so Python runtime will never throw an error. (Because it is not evaluated forever )
We did same thing for PIL.Image. But, it works fine without gemini instlation option.

https://github.com/Mirascope/mirascope/blob/19a6b76e487c9e2c723be4713ec49cca7c6862b8/mirascope/core/base/_utils/_convert_messages_to_message_params.py#L18-L22